src/Compilers/CSharp/Portable/Binder/BinderFactory.BinderFactoryVisitor.cs

```diff
@@ -679,8 +679,11 @@ private Binder VisitTypeDeclarationCore(TypeDeclarationSyntax parent)
                 // and position is more likely to be in the body, so lets check for "inBody" first.
                 if (parent.OpenBraceToken != default &&
                     parent.CloseBraceToken != default &&
-                    (LookupPosition.IsBetweenTokens(_position, parent.OpenBraceToken, parent.CloseBraceToken) ||
-                     LookupPosition.IsInAttributeSpecification(_position, parent.AttributeLists)))
+                    LookupPosition.IsBetweenTokens(_position, parent.OpenBraceToken, parent.CloseBraceToken))
```

Comment:
>LookupPosition.IsBetweenTokens(_position, parent.OpenBraceToken, parent.CloseBraceToken) [](http://example.com/codeflow?start=20&length=88)

It doesn't look like there is an intent to make a behavior change here.  Consider reverting.

Reply:
This is an intentional change. The behavior within attribute on type should not depend on whether the type declaration has braces. There was a bug with records around that (https://github.com/dotnet/roslyn/issues/60379).

Reply:
I can make a separate PR for this part of the fix if you prefer.